File: java/socketd/src/main/java/org/noear/socketd/core/HeartbeatHandler.java
package org.noear.socketd.core;

import java.io.IOException;

/**
 * 心跳处理器
 *
 * @author noear
 * @since 2.0
 */
public interface HeartbeatHandler {
    /**
     * 心跳处理
     */
    void heartbeatHandle(Session session) throws Exception;
}

File: javascript/ts/org/noear/socketd/transport/core/HeartbeatHandler.ts
import {Session} from "./Session";

/**
 * 心跳处理器
 *
 * @author noear
 * @since 2.0
 * @class
 */
export interface HeartbeatHandler {
    /**
     * 心跳处理
     * @param {*} session
     */
    (session: Session);
}